nautobot/utilities/utils.py

```diff
@@ -977,29 +988,32 @@ def convert_querydict_to_factory_formset_acceptable_querydict(request_querydict,
     num = 0
     request_querydict = request_querydict.copy()
     request_querydict.pop("q", None)
-    for lookup_type, value in request_querydict.items():
+    for filter_field_name, value in request_querydict.items():
         # Discard fields without values
         if value:
-            if lookup_type in filterset_class_fields:
-                lookup_field = re.sub(r"__\w+", "", lookup_type)
-                lookup_value = request_querydict.getlist(lookup_type)
+            if filter_field_name in filterset_class_fields:
+                if isinstance(filterset.filters[filter_field_name], RelationshipFilter):
```

Comment:
Another case where we might be able to use `hasattr(..., "relationship")` to avoid the need for the inner import?